closed/autoconf/custom-hook.m4

```diff
@@ -308,8 +308,7 @@ AC_DEFUN_ONCE([OPENJ9_PLATFORM_SETUP],
 AC_DEFUN_ONCE([OPENJDK_VERSION_DETAILS],
 [
   OPENJDK_SHA=`git -C $TOPDIR rev-parse --short HEAD`
-
-  # Find first rev tagged by jdk-12* but not containing "_openj9"
+  # Find first rev tagged by jdk-11* but not containing "_openj9"
```

Comment:
The comment still needs to be fixed.

Reply:
i.e. not changed from the original. It should say jdk-12 not jdk-11

Reply:
Fixed